glotaran/io/register.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import annotations
```

Comment:
This whole file looks and feels like a huge copypasta orgy and could probably be a lot simplified, by some decorator magic.

Reply:
yes, it is basically mostly convenience wrappers for users. Feel free to refactor in another commit.